### PR TITLE
Add v vls support

### PR DESCRIPTION
**Pull request by remimimimi**

### CODE REVIEW

File: lsp-mode.el [hunk 1]
```diff
@@ -756,6 +756,7 @@ Changes take effect only when a new session is started."
                                         (csharp-tree-sitter-mode . "csharp")
                                         (plain-tex-mode . "plaintex")
                                         (latex-mode . "latex")
+                                        (v-mode, "v")
```
Comment (yyoncho) [inline lsp-mode.el:759]: . not ,

File: clients/lsp-v.el [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+Leave as just the executable name to use the default behavior of
+finding the executable with variable `exec-path'."
+  :group 'lsp-v
+  :type 'string)
+
+(lsp-register-client
+ (make-lsp-client
+  :new-connection (lsp-stdio-connection (lambda () lsp-v-vls-executable))
+  :activation-fn (lsp-activate-on "v")
+  :server-id 'vls))
```
Comment (yyoncho) [inline clients/lsp-v.el:45]: vls is taken by vetur laguage server, maybe `v-ls`?

**Review by yyoncho — COMMENTED** [review]
Looks good, some nits.

**Review by yyoncho — APPROVED** [review]

**Comment by remimimimi** [comment]
Is this CI fault with macos targets? @yyoncho

**Comment by yyoncho** [comment]
> Is this CI fault with macos targets? @yyoncho

It is fine, melpa package installation fails from time to time.

Thank you for contributing to `lsp-mode`!

**Pull request merged**